### PR TITLE
execdetails: unify WRU/RRU to RU

### LINKED ISSUE #43187: explain analyze only report ru
## Enhancement
Currently, we only introduce one RU unit to the user, so unify the rru, wru in the explain analyze report.

### PR DESCRIPTION
**Pull request by nolouch**
### What problem does this PR solve?

Issue Number: close #43187

Problem Summary:

Currently, we only introduce one RU unit to the user, so unify the rru, wru in the explain analyze report.

wait https://github.com/tikv/client-go/pull/769
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
image

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by glorv — APPROVED** [review]
LGTM

**Review by JmPotato — APPROVED** [review]

**Comment by glorv** [comment]
depending on https://github.com/tikv/client-go/pull/769

File: DEPS.bzl [hunk 1]
```diff
@@ -4132,8 +4132,8 @@ def go_deps():
         name = "com_github_tikv_client_go_v2",
         build_file_proto_mode = "disable_global",
         importpath = "github.com/tikv/client-go/v2",
-        sum = "h1:pfDrSVAnfkk2EkrOc0iOmtA4n8F6TL9oEAK8R/enC50=",
```
Comment (hawkingrei) [inline DEPS.bzl:4135]: Please fix the conflict.

Comment (hawkingrei) [reply DEPS.bzl:4135]: BTW, https://github.com/pingcap/tidb/pull/43225 is also updating the dependencies. Can you upgrade to the latest version?

Comment (you06) [reply DEPS.bzl]: Upgraded by #43208

**Pull request merged**